.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,89 @@
+---
+name: Windows build and test
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - '*'
+  release:
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+      matrix:
+        include: [
+          { arch: x86_64, msystem: mingw64, tests: none },
+          { arch: x86_64, msystem: mingw64, tests: main },
+          { arch: i686, msystem: mingw32, tests: main },
+          { arch: i686, msystem: mingw32, tests: none },
+        ]
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+        with:
+          path: ${{ github.event.repository.name }}
+      - name: Fetch Data-Sample repository
+        uses: actions/checkout@v2
+        with:
+          repository: coin-or-tools/Data-Sample
+          path: Data/Sample
+      - name: Fetch ThirdParty-Glpk repository
+        uses: actions/checkout@v2
+        with:
+          repository: coin-or-tools/ThirdParty-Glpk
+          path: ThirdParty/Glpk
+      - name: Checkout coinbrew
+        uses: actions/checkout@v2
+        with:
+          repository: coin-or/coinbrew
+          path: coinbrew
+      - name: Set correct host flag and install requirements
+        run: |
+          echo "host_flag=--host=${{ matrix.arch }}-w64-mingw32" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+          C:\msys64\usr\bin\pacman -S mingw-w64-${{ matrix.arch }}-lapack mingw-w64-${{ matrix.arch }}-winpthreads-git mingw-w64-${{ matrix.arch }}-readline mingw-w64-${{ matrix.arch }}-suitesparse mingw-w64-${{ matrix.arch }}-metis --noconfirm
+      - name: Set up msys with ${{ matrix.msystem }}
+        uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          install: >-
+            base-devel
+            git
+          path-type: inherit
+          msystem: ${{ matrix.msystem }}
+      - name: Build project
+        run: |
+          [[ ${{ matrix.tests }} == "none" ]] && debug="--enable-debug" suffix="-dbg" || debug="" suffix=""
+          ./coinbrew/coinbrew build ${{ github.event.repository.name }} --no-prompt --skip='ThirdParty/Metis ThirdParty/Mumps' --build=x86_64-w64-mingw32 ${{ env.host_flag }} $debug --verbosity 4 --tests ${{ matrix.tests }} --enable-relocatable
+          cp ${{ github.event.repository.name }}/README.md dist/
+          cp ${{ github.event.repository.name }}/LICENSE dist/
+          tar -czvf release.tar.gz -C dist .
```

Comment:
The winDos people may be more familiar with a zip than a tarball.

Reply:
I would do a tarball for the g++ builds, even in MSyS and use zip only for the Visual Studio builds, if we ever get there :).

Reply:
I already changed it to zip, and since that is currently how we are packaging all windows builds it makes sense to me to keep it that way. I opened and issue if you want to discuss this further: #166